app/src/main/res/layout/fragment_participant_flow_matching.xml

```diff
@@ -102,7 +102,7 @@
             android:layout_marginEnd="32dp"
             android:layout_marginBottom="16dp"
             android:textAppearance="@style/ButtonTextStyling"
-            android:backgroundTint="@color/colorTextOnLight"
+            android:backgroundTint="@color/colorAccent"
```

Comment:
Hey @Vinald  is this ticket ready for review, i see youve closed the other PR  that handles  validation of match or register page.